docs/ref/settings.txt

```diff
@@ -428,6 +428,22 @@ any hyphens with underscores, and adding an ``'HTTP_'`` prefix to the name.
 For example, if your client sends a ``'X-XSRF-TOKEN'`` header, the setting
 should be ``'HTTP_X_XSRF_TOKEN'``.
 
+.. setting:: CSRF_TRUSTED_ORIGINS
+
+CSRF_TRUSTED_ORIGINS
+--------------------
+
+.. versionadded:: 1.9
+
+Default: ``[]`` (Empty list)
+
+A list of hosts which are trusted origins for unsafe (e.g. `POST`) requests.
+By default, Django's CSRF protection requires that under HTTPS an unsafe
+request must come from your same domain. If you need cross-origin unsafe
+requests over HTTPS, include the source hosts in this setting. Don't include
+any host in this setting that you don't control or trust to not CSRF your
```

Comment:
"not to commit a CSRF attack against your site"? (is CSRF often used as a verb?)